### PR TITLE
TC_17.04_002 | Move Organization Folder into Folder

### PR DESCRIPTION
**Pull request by SiarheiMSV**
https://trello.com/c/oMY1Lvbw/1129-at1704002-organization-folder-move-organization-folder-into-folder

### CODE REVIEW

**Comment by DariaMas7** [comment]
We should include in our commit AT not TC

File: cypress/e2e/organizationFolderMove.cy.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+}
+
+function createFolder1(folderNew) {
+    cy.get('a[href="/view/all/newJob"]').click();
+    cy.get('#name').type(createFolder.folderName);
+    cy.get('.com_cloudbees_hudson_plugins_folder_Folder').click();
+    cy.get('#ok-button').click();
+    cy.get('button[name="Submit"]').click();
+    cy.get('#breadcrumbBar li:first-child').click();
+}
```
Comment (DariaMas7) [inline cypress/e2e/organizationFolderMove.cy.js:24]: Also, it's a good practice to put all code regarding creating folders in beforeEach

File: cypress/e2e/organizationFolderMove.cy.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+
+describe('orgFolderConfigurate', () => {
```
Comment (DariaMas7) [inline cypress/e2e/organizationFolderMove.cy.js:26]: Describe is supposed to be the same as your spec name

File: cypress/e2e/organizationFolderMove.cy.js [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    
+    it('TC_17.04_002 | Move Organization Folder into Folder', () => {
```
Comment (DariaMas7) [inline cypress/e2e/organizationFolderMove.cy.js:28]: in it() we write the name of AT

File: cypress/e2e/organizationFolderMove.cy.js [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+        createFolder1()
+        createOrgFolder()
+        let linkBefore = (`http://localhost:${PORT}/job/${orgFolderConfigure.orgFolderName}/`)
+        cy.get(`a[href="job/${orgFolderConfigure.orgFolderName}/"]`).click()
+        cy.get('#breadcrumbBar').should('contain',`${orgFolderConfigure.orgFolderName}`)
```
Comment (DariaMas7) [inline cypress/e2e/organizationFolderMove.cy.js:33]: there is no need to use ${}
we can simple say .should('contain',orgFolderConfigure.orgFolderName)

File: cypress/e2e/organizationFolderMove.cy.js [hunk 5]
```diff
@@ -0,0 +1,48 @@
...
+        cy.get('#jenkins-home-link').click()
+        cy.get(`a[href="job/${orgFolderConfigure.orgFolderName}/"] .jenkins-menu-dropdown-chevron`).click({force: true})
+        cy.get(`a[href="/job/${orgFolderConfigure.orgFolderName}/move"]`).click()
+        cy.get('select').select(`Jenkins » ${createFolder.folderName}`).should('have.value', '/TestFolder')
```
Comment (DariaMas7) [inline cypress/e2e/organizationFolderMove.cy.js:37]: /TestFolder might be placed in fixtures

File: cypress/e2e/organizationFolderMove.cy.js [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+        cy.get('button[name="Submit"]').click()
+        cy.get('#jenkins-home-link').click()
+        cy.get(`a[href="job/${createFolder.folderName}/"]`).click()
+        cy.get(`a[href="job/${orgFolderConfigure.orgFolderName}/"]`).click()
+        cy.url().should('include', `http://localhost:${PORT}/job/${createFolder.folderName}/job/${orgFolderConfigure.orgFolderName}/`)
+        .and('not.deep.equal', 'linkBefore')
+    });
+    
+
```
Comment (DariaMas7) [inline cypress/e2e/organizationFolderMove.cy.js:46]: its not necessary to leave those empty lines

File: cypress/e2e/organizationFolderMove.cy.js (hunk 2 above)
Comment (SiarheiMSV) [reply cypress/e2e/organizationFolderMove.cy.js]: Done

File: cypress/e2e/organizationFolderMove.cy.js (hunk 3 above)
Comment (SiarheiMSV) [reply cypress/e2e/organizationFolderMove.cy.js]: done

File: cypress/e2e/organizationFolderMove.cy.js (hunk 5 above)
Comment (SiarheiMSV) [reply cypress/e2e/organizationFolderMove.cy.js]: done(code changed)

File: cypress/e2e/organizationFolderMove.cy.js (hunk 1 above)
Comment (SiarheiMSV) [reply cypress/e2e/organizationFolderMove.cy.js]: When i put unctions in beforeEach code is not works

File: cypress/e2e/organizationFolderMove.cy.js (hunk 5 above)
Comment (SiarheiMSV) [reply cypress/e2e/organizationFolderMove.cy.js]: Thank you for your comments and time!!!!

**Review by shablouskaya — APPROVED** [review]

**Review by IharStankevich — APPROVED** [review]

**Pull request merged**